master/buildbot/status/web/console.py

```diff
@@ -404,21 +397,21 @@ def displayStatusLine(self, builderList, allBuilds, revision, debugInfo):
 
         details = []
         nbSlaves = 0
-        for category in builderList:
-            nbSlaves += len(builderList[category])
+        for tag in builderList:
+            nbSlaves += len(builderList[tag])
 
-        # Sort the categories.
-        categories = sorted(builderList.keys())
+        # Sort the tags.
+        tags = sorted(builderList.keys())
 
         builds = {}
 
-        # Display the boxes by category group.
-        for category in categories:
+        # Display the boxes by tag group.
+        for tag in tags:
```

Comment:
Inside this `for..` loop there was a variable `tag` used, which overwrites the value assigned here.